core/orchestrator.go

```diff
@@ -630,19 +634,24 @@ func (n *LivepeerNode) transcodeFrames(ctx context.Context, sessionID string, ur
 
 	// We only support base64 png data urls right now
 	// We will want to support HTTP and file urls later on as well
-	dirPath := path.Join(n.WorkDir, "input", sessionID+"_"+string(RandomManifestID()))
+	dirPath := path.Join(WorkDir, "input", sessionID+"_"+string(RandomManifestID()))
```

Comment:
Was this change intentional?  I don't see WorkDir defined in this scope.

Reply:
it looks like a typo i missed while testing locally changing the WorkDir to local directory and reverting it back to WorkDir without the prefix n. didnt throw an error while final testing, so i didnt catch it. Just tested it again with live-portrait and image-to-video just be sure and yeah it didnt show any issue.

Reply:
changed it back to the original after testing it again with the pipelines `live-portrait and image-to-video`